app/views/repositories/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<% title "GitHub Users" %>
```

Comment:
Currently submitting this form will redirect to `create.html.erb` which is empty. The `RepositoriesController` will populate either a flash notification or alert.